modules/api/pkg/provider/kubernetes/member.go

```diff
@@ -293,7 +292,7 @@ func (p *ProjectMemberProvider) MapUserToRoles(ctx context.Context, user *kuberm
 	}
 
 	if user.Spec.IsGlobalViewer {
-		roles.Insert(handler.ViewersRole)
+		roles.Insert("viewers")
```

Comment:
Instead of moving away from constants, move https://github.com/kubermatic/dashboard/blob/main/modules/api/pkg/ee/group-project-binding/handler/handler.go#L47-L51 somewhere else so that they are accessible for CE/EE both.